canonical-websites/www.canonical.com
# Initial Vanilla v1 pass on /index, /services, /products

## Done

Covert /index, /products & /services to Vanilla v1

## QA

- Pull code
- Sanity check code
- Run `./run serve --watch`
- Check pages:

http://0.0.0.0:8002/
http://0.0.0.0:8002/products
http://0.0.0.0:8002/services
http://0.0.0.0:8002/services/contact-us
http://0.0.0.0:8002/services/thank-you

## Issue / Card

Related #144
